### PR TITLE
Update CODEOWNERS for benefits management tools team

### PR DESCRIPTION
**Pull request by jerekshoe**
**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be completed as written. Provide known gaps to testing that may raise the risk of merging to production.

## Summary
Update CODEOWNERS for benefits management tools team. Benefits Team 1 was split into two teams (benefits-management-tools and benefits-decision-reviews) with new GH teams for both (benefits-management-tools-be and benefits-decision-reviews-be respectively). This is one of multiple planned PRs to start moving ownership to the new team for benefits-management-tools specifically.

## Related issue(s)
- department-of-veterans-affairs/va.gov-team#65783

## What areas of the site does it impact?
*(Describe what parts of the site are impacted and*if*code touched other areas)*

## Acceptance criteria

- [ ]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [ ]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

## Requested Feedback

(OPTIONAL)_What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1260,7 +1261,7 @@ spec/lib/carma @department-of-veterans-affairs/vfs-10-10 @department-of-veterans
 spec/lib/caseflow @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
 spec/lib/central_mail @department-of-veterans-affairs/lighthouse-banana-peels
 spec/lib/chip @department-of-veterans-affairs/vsa-healthcare-health-quest-1-backend @department-of-veterans-affairs/patient-check-in @department-of-veterans-affairs/backend-review-group
-spec/lib/claim_status_tool @department-of-veterans-affairs/Benefits-Team-1 @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
+spec/lib/claim_status_tool @department-of-veterans-affairs/benefit-management-tools-be @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
```
Comment (rmtolmach) [inline .github/CODEOWNERS:1264]: The missing `s` at the end of "benefit" is causing the error. 
```suggestion
spec/lib/claim_status_tool @department-of-veterans-affairs/benefits-management-tools-be @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
```

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1298,9 +1299,8 @@ spec/lib/inherited_proofing @department-of-veterans-affairs/vsp-identity
 spec/lib/json_schema @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
 spec/lib/lgy @department-of-veterans-affairs/vsa-ebenefits @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
 spec/lib/lighthouse @department-of-veterans-affairs/lighthouse-banana-peels
-spec/lib/lighthouse/auth @department-of-veterans-affairs/lighthouse-banana-peels
-spec/lib/lighthouse/benefits_claims @department-of-veterans-affairs/lighthouse-banana-peels
-spec/lib/lighthouse/benefits_claims/service_spec.rb @department-of-veterans-affairs/lighthouse-banana-peels @department-of-veterans-affairs/Benefits-Team-1
+spec/lib/lighthouse/auth @department-of-veterans-affairs/benefit-management-tools-be @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
+spec/lib/lighthouse/benefits_claims @department-of-veterans-affairs/benefit-management-tools-be @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
```
Comment (rmtolmach) [inline .github/CODEOWNERS:1303]:
```suggestion
spec/lib/lighthouse/benefits_claims @department-of-veterans-affairs/benefits-management-tools-be @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
```

File: .github/CODEOWNERS (hunk 1 above)
Comment (jerekshoe) [reply .github/CODEOWNERS:1264]: Whoops, I thought the danger check would catch these. Thanks!

**Comment by rmtolmach** [comment]
Thanks for helping get CODEOWNERS up-to-date, @jerekshoe!

I noticed there were a number of instances where you removed @department-of-veterans-affairs/lighthouse-banana-peels as reviewers, so I added them as reviewers.

**Review by kristen-brown — APPROVED** [review]

**Review by rmtolmach — APPROVED** [review]
LGTM! Thanks for the review for Lighthouse, @kristen-brown!

**Pull request merged**